spec/rails_helper.rb

```diff
@@ -9,6 +9,8 @@
 require "selenium-webdriver"
 require "view_component/test_helpers"
 
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```

Comment:
If we're adding this here we should probably remove the explicit requires from individual specs that use this?